conans/model/info.py

```diff
@@ -367,9 +370,9 @@ def dumps(self):
         if self.build_requires:
             result.append("[build_requires]")
             result.append(self.build_requires.dumps())
-        conf_dumps = self.conf.dumps()
-        if conf_dumps:
-            # result.append("[conf]")  # Let's do it when apply FIXME above
+        if self.conf:
+            # TODO: Think about the serialization of Conf, not 100% sure if dumps() is the best
+            result.append("[conf]")
```

Comment:
I'm not sure what you mean by that `TODO`

Reply:
Conf serializes "Profile-like". Not sure if that is the best serialization to "conaninfo.txt" or we could want to serialize final values instead